### ISSUE TITLE
Connect Contact form to API

### ISSUE THREAD

**Issue by stephanieguzm** [stephanieguzm/portfolio#15]

**Comment by stephanieguzm** [comment]
Troubleshoot error in terminal
```
- error TypeError: Cannot read properties of undefined (reading 'headers')
    at eval (webpack-internal:///(sc_server)/./node_modules/next/dist/server/future/route-modules/app-route/module.js:277:61)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

```

**Comment by stephanieguzm** [comment]
I'm successfully receiving a confirmation email with my test message and my however I continued to get the above message in my CLI. Bug resolved with commit [93211a1](https://github.com/stephanieguzm/portfolio/commit/93211a12f8495fc944d70121e5e1627c942192c4)

Successfully receiving info on form and forwarding to email client.
